spec/support/spec/platform_data.rb

```diff
@@ -56,6 +56,12 @@ def platform_data(p, d)
       config_path: '/opt/app/patroni/etc/postgresql.yml',
     },
   }
+  sanitized_platform = if %r{(RedHat|CentOS|AlmaLinux|Rocky|OracleLinux)-(\d+)}.match(p)
+                         "RedHat-#{Regexp.last_match(2)}"
+                       else
+                         p
+                       end
+  puts sanitized_platform
```

Comment:
Debug code?

Reply:
cleaned it up in https://github.com/voxpupuli/puppet-patroni/pull/71